Assets/Scenes/Level2.unity

```diff
@@ -108379,13 +108379,21 @@ PrefabInstance:
     serializedVersion: 3
     m_TransformParent: {fileID: 1111688093}
     m_Modifications:
+    - target: {fileID: 1395949811245891063, guid: 0fedb40c7675c7a40b7785be1f0863bf, type: 3}
+      propertyPath: m_Size.y
+      value: 1.9892833
+      objectReference: {fileID: 0}
+    - target: {fileID: 1395949811245891063, guid: 0fedb40c7675c7a40b7785be1f0863bf, type: 3}
+      propertyPath: m_Offset.y
+      value: -0.5529442
+      objectReference: {fileID: 0}
```

Comment:
hm why do we have override values for the capsule collider on knitby here? can we just remove them?

Reply:
oops thx these were meant to be applied